tests/behat/uploadFileToMoodle.feature

```diff
@@ -0,0 +1,27 @@
+@ocis @javascript
+Feature: list the resources
+  As a user
+  I want to list the resources
```

Comment:
These words do not match with the name of the feature file.

Will this be a test of a "list resources" feature, or an "upload file" feature?